packages/app/src/cli/models/extensions/specifications/editor_extension_collection.ts

```diff
@@ -17,17 +18,33 @@ const EditorExtensionCollectionSchema = BaseSchema.extend({
   type: zod.literal('editor_extension_collection'),
 })
 
+export type EditorExtensionCollectionType = zod.infer<typeof EditorExtensionCollectionSchema>
+
 const editorExtensionCollectionSpecification = createExtensionSpecification({
   identifier: 'editor_extension_collection',
   schema: EditorExtensionCollectionSchema,
   appModuleFeatures: (_) => [],
+  validate: async (config, path) => {
+    const errors: string[] = []
+    const inCollection = makeExtensionsInCollection(config)
+
+    if (!config.handle) {
+      return err(`Editor extension collection with name ${config.name} must have a handle`)
+    }
+
+    if (inCollection.length < 2) {
+      errors.push(`There must be at least two extensions in editor extension collection ${config.handle}`)
+    }
+
```

Comment:
Is there any way to validate the extension handles that are included here? If we can't here we definitely will need to in core.

Reply:
No we cant do this here. I added validation for that already here https://github.com/Shopify/cli/pull/3728/files#diff-cf7257d7c0c4111afc303b94c039388faf0f27ff4e121ce30f89db3d0c9f2d10R377

Reply:
Ah okay. I was just wondering.